nipc-openapi/NIPC_REST.yaml

```
@@ -112,9 +100,9 @@ paths:
       parameters:
         - name: id
           in: query
-          description: device ids that need to be filtered
+          description: device or group id that need to be filtered
           required: false
-          explode: true
+          explode: false
           schema:
             type: string
             format: uuid
```

For a comma separated list, the type must be array and not string. 
```suggestion
            type: array
            items:
              type: string
              format: uuid
```